### PR TITLE
Update ACS Version

### PR DESCRIPTION
**Pull request by stuft2**

### CODE REVIEW

**Review by blakemorgan — DISMISSED** [review]

**Review by GaryGSC — DISMISSED** [review]
Simple enough.

[Release notes for `terraform-aws-acs-info` v3.2.0](https://github.com/byu-oit/terraform-aws-acs-info/releases/tag/v3.2.0)

**Comment by stuft2** [comment]
@blakemorgan @GaryGSC I updated the ci to test versions 0.12 -> 1.0 of terraform so we get a more accurate view of what versions of terraform we support. I also updated the readme to show the new version of this module that we'll publish when this PR is merged.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,15 +15,33 @@ jobs:
...
+                "aws_key_name":"byu_oit_terraform_dev_key",
+                "aws_secret_name":"byu_oit_terraform_dev_secret"
+              },
               {
                 "tf_version":"0.13.2",
-                "tf_working_dir":"./examples/ci-13",
+                "tf_working_dir":"./examples/ci-0_13",
+                "aws_key_name":"byu_oit_terraform_dev_key",
+                "aws_secret_name":"byu_oit_terraform_dev_secret"
+              }
```
Comment (GaryGSC) [inline .github/workflows/ci.yml]:
```suggestion
              },
```

**Review by GaryGSC — APPROVED** [review]
Thanks for expanding the CI matrix.

I think this is a strict improvement over what we had previously, so even with the failing CI, this LGTM.

**Pull request merged**